lib/venue.rb

```diff
@@ -1,2 +1,40 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+
+    @patrons = []
+  end
+
+
+  def add_patron(patron)
+    patrons << patron
+  end
+
+
+  def yell_at_patrons
+  upcase_patrons = []
```

Comment:
Great job iterating through the `@patrons` array and creating an accumulator to capture what you are wanting with the `.upcase`